### DISCUSSION TITLE
Is there a mapping between the NAICS codes used in v1.1.1 and v1.2.?

### DISCUSSION THREAD

**Discussion by isabelvelasco1958** [USEPA/supply-chain-factors#81] [General]
Is there a mapping between the NAICS codes used in v1.1.1 and v1.2.?

**Comment by WesIngwersen** [comment]
@bl-young

**Comment by bl-young** [comment]
v1.1.1 produced factors based on BEA codes. In v1.2, the factors are based on NAICS codes. In both cases, the underlying USEEIO model is still based on BEA data (~400 sectors). You'll notice in the v1.2 files, you'll notice there is a field `Reference USEEIO Code`. This field shows the underlying BEA/USEEIO code used to produce the factor which aligns with aligns with the specific NAICS.

**Reply by isabelvelasco1958** [reply]
Wow ok thank you very much! I may have more questions, but goofd for now :)
